src/validate.ts

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+const Ajv = require('ajv'); 
```

Comment:
I think we can remove the AJV and Chai imports in this pull request and do them in a follow-up.